### ISSUE TITLE
FRONT DOOR: Create main functionalities section.

### ISSUE THREAD

**Issue by crownz** [serveuslt/seo#13]
* This section will go straight after header and form.
* It should have eye catching illustrations with well visible text to navigate user to the core features of the application.

**Comment by mantasak** [comment]
Not sure if this issue is required @crownz?

Possibly covered by [seo/18](https://github.com/serveuslt/seo/issues/18).

**Comment by crownz** [comment]
@mantasak agree, was initial idea but it seems it looks to dark/heavy for our current implementation. Putting this on hold.
